### PR TITLE
Use wheel event only in mouseWheelInput.js

### PR DESCRIPTION
**Pull request by talkdirty**
Previously, both mousewheel and DOMMouseScroll Events were used, the
latter of which is firefox specific. DOMMouseScroll shows problems when
two finger scrolling on Laptops. Newer browser versions all use the
standardized wheel event now, which should be the right thing to use
here.

* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

See [this issue](https://github.com/cornerstonejs/cornerstoneTools/issues/643)

### CODE REVIEW

**Comment by talkdirty** [comment]
@dannyrb

**Comment by dannyrb** [comment]
@talkdirty, I found a good resource for which versions of which browser vendors use which scroll events. It was included with a snippet of code to help normalize the event handling across browsers. I'll revisit this PR now that I have that information. I'll likely update our listener to normalize the event and well :+1:

**Comment by talkdirty** [comment]
That's great, thank you! Out of interest, could you link the resource you
found? Also let me know if you require any assistance for this.
Georg

**Comment by dannyrb** [comment]
https://github.com/facebookarchive/fixed-data-table/blob/master/src/vendor_upstream/dom/normalizeWheel.js

**Comment by dannyrb** [comment]
@talkdirty, alright. I've merged the fix for this to `vNext`. We have the option of backporting this to `v2`, but we would lose some browser support because I dropped listening for `'mousewheel', 'DOMMouseScroll'` and instead only listen to `wheel`. I'm okay with doing that in `vNext`, because it's the next major release, and I'm okay with the breaking change there.

Whether we should do this for `v2` isn't as cut and dry, even though I really, really hope no one is using `cornerstone` on `<= IE 9` or older versions of evergreen browsers. I would be willing to backport it if @swederik gave the 👍

I believe #643 can be addressed with a non-breaking change, but I don't have the bandwidth to look into it.


*tl;dr:*

- I will look at and merge any non-breaking solutions
- If @swederik gives a 👍, I would spend the time to merge the `vNext` solution to this branch
- I recommend waiting for `vNext` RC or bravely porting existing code to it now if this issue is plaguing you

I'll close this PR for now; feel free to continue discussion here, or on the related issue
